src/rabbit_vhost.erl

```diff
@@ -185,3 +197,6 @@ info_all(Ref, AggregatorPid)        -> info_all(?INFO_KEYS, Ref, AggregatorPid).
 info_all(Items, Ref, AggregatorPid) ->
     rabbit_control_misc:emitting_map(
        AggregatorPid, Ref, fun(VHost) -> info(VHost, Items) end, list()).
+
+dir(Vhost) ->
+    base64:encode(Vhost).
```

Comment:
After a brief internal discussion we decided it's not safe to use Base64 or Base32 for our needs because they both use lower and upper case characters. Queue directory naming uses MD5 hashing, which is safe but annoying for operators as it's not easy to infer what directory corresponds to what queue.

I suspect we can create a text file in each vhost (and possibly queue) directory that contains its actual name.